### PR TITLE
add render to G2

### PR DESCRIPTION
**Pull request by Waloncycler**
test coverage *%

### CODE REVIEW

File: test/MultiplicationTable.test.ts [hunk 1]
```diff
@@ -9,8 +9,16 @@ describe('Multiplication Table', () => {
 
     //when
```
Comment (wenruiLiA15) [inline test/MultiplicationTable.test.ts:10]: test case better match with task diagram

File: src/MultiplicationTable.ts [hunk 2]
```diff
@@ -1,5 +1,40 @@
...
+    }
+    var multiplicationTable:Array<string> = []
+    for(let i = start; i <= end; i++) {
+      let resultString = this.multiplicationLineGenerator(start, i)
+      multiplicationTable.push(resultString)
+    }
+    return multiplicationTable.toString()
+  }
+  
+  public isLegalRange(start: number, end: number): boolean{
```
Comment (BOBOPOS) [inline src/MultiplicationTable.ts]: Maybe input decimals and the whole program still works

Comment (wenruiLiA15) [inline src/MultiplicationTable.ts]: if method not directly tested, better use private method

**Review by wenruiLiA15 — COMMENTED** [review]
need more tests

**Comment by nnnancyyy** [comment]
I feel if given (1,2) the output is not as expected 
`1*1=1  \n1*2=2  2*2=4`

**Pull request closed without merging**